components/Item/__tests__/Item.test.js

```diff
@@ -6,7 +6,10 @@ import Item from '../Item';
 describe('<Item />', () => {
   it('renders correctly', () => {
     const wrapper = mount(<Item name="Item one" id="123" />);
-
     expect(wrapper).toMatchSnapshot();
   });
+  it('should contain name=testing', () => {
```

Comment:
The snapshot test will take care of this.

So general policy is that 

To test if markup is coming correctly, rely on snapshots

To test behavior (onClick, etc) use simulations.